lib/tasks/production-copy.rake

```diff
@@ -205,6 +205,11 @@ module CloneProductionHelper
     configs["development"]["database"]
   end
 
+  def local_development_db_port
+    configs = YAML.load_file(File.new("config/database.yml"))
+    configs["development"]["port"]
```

Comment:
Why do you need the port? Are you running on a non-standard port? Why?

Reply:
To run OFN and LO locally they have different postgres requirements so I'm running multiple versions that each run on their own port. https://medium.com/keeping-code/running-multiple-postgresql-versions-simultaneously-on-macos-linux-90b3d7e08ffd

Reply:
Let's discuss in our next call.

Reply:
Would be good to get this into master or refactored to be nicer. I keep having to copy this fix to my feature branches.

Reply:
Do you time today for a quick chat?